tests/e2e/features/artifacts/transferPage/artifacts.feature

```diff
@@ -0,0 +1,77 @@
+@artifacts @regression @transferPage
+Feature: Artifacts - UI
+
+  Background:
+    Given Connect Metamask extension with login action
+
+
+  @id1646:I @tokens @testnet
+  Scenario: Check artifacts on tokens dropdown on Transfer page (Testnet)
+    Given I go to page "/transaction/zksync/era/send?network=era-goerli&address=0x47BCD42B8545c23031E9918c3D823Be4100D4e87"
+    Then I click by "testId" with "token-dropDown" value
+    Then Element with "text" "Choose token" should be "visible"
+    Then Element with "placeholder" "Symbol or address" should be "visible"
+    Then Element with "class" "token-balance-amount" should be "visible"
+    Then Element with "class" "token-balance-price" should be "visible"
+
+  @id1646:II @tokens @testnet
+  Scenario Outline: Check artifacts on tokens dropdown on Transfer page (Testnet)
+    Given I go to page "/transaction/zksync/era/send?network=era-goerli&address=0x47BCD42B8545c23031E9918c3D823Be4100D4e87"
+    Then I click by "testId" with "token-dropDown" value
+    Then Element with "text" "<TokenName>" should be "visible"
+    Then Element with "partial src" "<img>" should be "visible"
+    Then Element with "text" "<TokenAddress>" should be "visible"
+
+    Examples:
+      | TokenName | TokenAddress   | img      |
+      | ETH       | 0x000000...00A | eth.svg  |
+      | DAI       | 0x3e7676...D4b | dai.svg  |
+      | LINK      | 0x406091...c78 | link.svg |
+      | USDC      | 0x0faF6d...2A9 | usdc.svg |
+      | wBTC      | 0x0BfcE1...e9c | wbtc.svg |
+
+  @id1645:I @tokens @mainnet
+  Scenario: Check artifacts on tokens dropdown on Transfer page (Mainnet)
+    Given I go to page "/transaction/zksync/era/send?network=era-mainnet&address=0x47BCD42B8545c23031E9918c3D823Be4100D4e87"
```

Comment:
This address is used to avoid extra steps for input of the address, click on it etc.
This address is ENS one, so available for all users - no extra data is provided.

Reply:
Are you sure that here we should use the address from the empty wallet? 0x47BCD42B8545c23031E9918c3D823Be4100D4e87

Reply:
It's not an empty wallet address - this is ENS foundation.eth.
I chose this one because it won't reveal our test wallets and available on ETH network.
<img width="1050" alt="image" src="https://github.com/matter-labs/dapp-portal/assets/118197764/a87e57b9-4a7e-43b1-830a-1d41e6968525">